Source/controls/controller_buttons.h

```diff
@@ -36,4 +37,114 @@ inline bool IsDPadButton(ControllerButton button)
 	    || button == ControllerButton_BUTTON_DPAD_RIGHT;
 }
 
+namespace controllerButtonIcon {
+using namespace std;

```

Comment:
What is the purpose of `using namespace std;` here? Seems like you're referencing `std::string` explicitly anyway.

Reply:
There is no reason for it to be here.